h/templates/client/annotation.html

```diff
@@ -6,46 +6,36 @@
   <header class="annotation-header">
     <!-- User -->
     <span ng-if="vm.user()">
-      <span>
-        <a class="annotation-user"
-           target="_blank"
-           ng-href="{{::vm.serviceUrl}}u/{{vm.user()}}"
-           >{{vm.username()}}</a>
-      </span>
-
-      <span class="annotation-collapsed-replies">
```

Comment:
If you have a stack of nested replies, there is no longer any indication that there are replies to a collapsed reply. Is that an intentional design decision?

eg. If you have:

```
Annotation
|-- Reply A
      |--- Reply B
```

And you collapse 'Reply A', that used to show as:
```
Annotation
|-- Reply A (1 reply)
```
I don't particularly mind the change, just need to be sure that it is intentional.